client/my-sites/themes/theme-showcase.jsx

```diff
@@ -695,31 +696,29 @@ class ThemeShowcase extends Component {
 								<div className="theme__search-container">
 									<div className="theme__search">
 										<div className="theme__search-input">
-											{ isSearchV2 ? (
-												<SearchThemesV2
-													query={ featureStringFilter + search }
-													onSearch={ this.doSearch }
-												/>
-											) : (
-												<SearchThemes
-													query={ filterString + search }
-													onSearch={ this.doSearch }
-													recordTracksEvent={ this.recordSearchThemesTracksEvent }
-												/>
-											) }
+											<SearchThemes
+												query={ isLoggedIn ? filterString + search : featureStringFilter + search }
+												onSearch={ this.doSearch }
+												recordTracksEvent={ this.recordSearchThemesTracksEvent }
+											/>
 										</div>
-										{ tabFilters && premiumThemesEnabled && ! isMultisite && (
-											<>
-												<SelectDropdown
-													className="section-nav-tabs__dropdown"
-													onSelect={ this.onTierSelectFilter }
-													selectedText={ translate( 'View: %s', {
-														args: getOptionLabel( tiers, tier ) || '',
-													} ) }
-													options={ tiers }
-													initialSelected={ tier }
-												></SelectDropdown>
-											</>
+										{ tabFilters && premiumThemesEnabled && ! isMultisite && ! isServerSide && (
```

Comment:
The `CustomSelectControl` doesn't support SSR, so I'll hide it for that case.

> The only downside of not rendering when SSR is that users will see those components "pop in," but IMO, it's an acceptable tradeoff.

More info: p4TIVU-32n-p2